doc/quickstart/configure.rst

```diff
@@ -479,58 +470,157 @@ speed with which the data can be read from and/or written to disk.
 
 .. _config-dask-default-scheduler:
 
-Dask default scheduler configuration
-------------------------------------
+Custom Dask default scheduler configuration
```

Comment:
```suggestion
Dask threaded scheduler configuration
```
To make it more specific?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2616/commits/f6fc37cf23c291066bc2f6fc6015860856c83c3c